providers/base/bin/nvidia_nvlink_check.sh

```diff
@@ -3,16 +3,16 @@
 set -e
 
 O=$(nvidia-smi nvlink -s)
-#Check NVLINK are avaliable first, if only 1 NVIDIA Graphic card, output will be empty.
-if [ -z "$O" ];then
-    echo "System does not support NVLINK or Only 1 NVIDIA graphic card installed";
+# If the system doesn't support NVLINK, the output will be None.
+if [ -z "$O" ]; then
+    echo "System does not support NVLINK."
     exit 1
-#If any inactive in output that means NVLINK not connected porpery or malfunction, -n use for verify output
-elif echo "$O" | grep -q inactive; then
-    echo "NVLINK either the bridge is missing/defective or not configured correctly";
+# If any inactive in output that means NVLINK not connected porpery or malfunction, -n use for verify output
```

Comment:
This is very difficult to read. I think better English would be:
"If the word "inactive" is found in the output, it means that NVLINK is not connected properly or it is malfunctioning."
I'd remove the reference to this '-n'.